hugo/layouts/shortcodes/citation_block.html

```diff
@@ -0,0 +1,18 @@
+{{ $year := now.Year }}
+{{ $date := now.Format "January 2, 2006" }}
+
+<!-- get the version number if prod build or set to X.X.X. if not -->
+{{ $version := partial "GetVersion.html" }}
+{{ $portalVersion := "X.X.X" }}
+{{ if $version.tag }}
+    {{ $portalVersion = index $version "tag" }}
+    {{ $portalVersion = replace $portalVersion "v" "" }}
+{{ end }}
+
```

Comment:
🥷 🐿️ 
```suggestion
{{ $portalVersion := $version.tag | default "X.X.X" | strings.TrimPrefix "v" }}
```

Reply:
Also, if the "v" is a hassle feel free to strip it in `GetVersion.html` (I think we don't use it for the JBrowse version in the footer)

Reply:
That was cool! I've add this in commit 3c2af9bc6cc2c83a5a87e87c4a6305c6db66ee5b (couldn't commit suggestion as this code has moved to a the render hooks file :) )